data/simulation/config.d.ts

```diff
@@ -39,10 +39,10 @@ export interface Config {
    * Only supported by Haskell simulation. */
   "leios-vote-send-recv-stages": boolean;
   /**
-   * Extends Leios so that EB producers include IBs directly from previous pipelines
-   * where no certified EB was observed.
-   * 
-   * Only supported by Rust simulation. */
+   * Extends Leios so that EB producers include IBs directly from previous pipelines.
+   * Due to casuality, the EB must always include them, even if those IBs end up being
+   * certified in their own pipeline.
+   */
```

Comment:
Taking another look at the extension, the behavior you describe looks correct. It's also the behavior that the rust simulation follows. I don't know why I wrote this comment.

Reply:
If it were IBs from more than 3 iterations, then we _could_ start making decisions based on whether we see the EB. However, it's not clear we'd want to do that even though we could.